part1.js

```diff
@@ -5,9 +5,34 @@
    dimensions of a box to be wrapped, for example:
      ['2x3x4', '3x5x8', '1x1x10']
 */
-function paperCalculator(boxes) {
-  // TODO: return the correct amount of wrapping paper needed.
-  return 0;
-};
 
-exports.paperCalculator = paperCalculator;
+var boxArray = ['2x3x4', '3x5x8', '1x1x10'];
+function paperCalculator(boxesArr) {
+  var total = 0;
+  for (i = 0; i < boxesArr.length; i++) {
+    total += paperCalculatorHelper(boxesArr[i]);
+  }
+  return total;
+}
+
+function paperCalculatorHelper(boxes) {
+  var l = boxes.slice(0, boxes.indexOf('x'));
+  var w = boxes.slice(boxes.indexOf('x') + 1, boxes.lastIndexOf('x'));
+  var h = boxes.slice(boxes.lastIndexOf('x') + 1);
+
+  var parL = parseInt(l, 10);
+  var parW = parseInt(w, 10);
+  var parH = parseInt(h, 10);
+
```

Comment:
This code works, but it can be done in fewer lines:
```javascript
var dimensions = box.split('x').map(Number);
var l = dimensions[0];
var w = dimensions[1];
var h = dimensions[2];
```